src/SolarEclipse2024.vue

```diff
@@ -4872,29 +4882,28 @@ video, #info-video {
 
 #speed-text {
   position: absolute;
-
   background-color: rgba(0, 0, 0, 0.5);
   padding-inline: 0.4em;
   padding-block: 0.15em;
   border-radius: 0.3em;
+  font-size: .75rem;
```

Comment:
This is a small thing, but for consistency, we should make this some fraction of the `--default-font-size` else it appears quite small on my desktop.